models/resnet.py

```diff
@@ -86,18 +86,22 @@ def __call__(self, x):
     return self.forward(x)
 
   def load_from_pretrained(self):
-    # TODO: replace with fake torch load
+    
```

Comment:
Why was this TODO removed?

Reply:
That was my mistake. Fixed in most recent commit